docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  mongo:
+    image: mongo
+    restart: always
+    ports: 
+      - 27017:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: toor
+      MONGO_INITDB_ROOT_PASSWORD: fabricateadmin
```

Comment:
Should probably be changed to labrinth or modbrinth admin, since we aren't called fabricate anymore

Reply:
It's really here to be a testing system